runtime/android/core/src/org/xwalk/core/XWalkActivity.java

```diff
@@ -108,25 +109,25 @@ public final void onMethodCallMissed(ReflectMethod method) {
     }
 
     @Override
-    public final void onXWalkLibraryStartupError(LibraryStatus status, Throwable e) {
```

Comment:
Could we implement the XWalkCoreListener in a subclass? since we don't want this mixed with other XWalkActivity callbacks or expose it to developers either.